tests/zpca/multiqc_data/multiqc_sources.txt

```diff
@@ -0,0 +1 @@
+Module	Section	Sample Name	Source
```

Comment:
I'm sorry, we worked on this some time ago and I forgot things.
What is the purpose of this file? Is this an input to the _zpca_?

Reply:
This is a file generated by `multiqc`. In fact, all the files in `multiqc_data` have been generated by `multiqc`, so stored it in `tests\` just like in other modules.